Lib/importlib/_bootstrap_external.py

```diff
@@ -140,13 +140,17 @@ def _path_isdir(path):
     return _path_is_mode_type(path, 0o040000)
 
 
-def _path_isabs(path):
-    """Replacement for os.path.isabs.
+if _MS_WINDOWS:
+    _pathsep_doubled = tuple(f'{s}{s}' for s in path_separators)
```

Comment:
This needs the mixed cases as well: `tuple(f'{s1}{s2}' for s1 in path_separators for s2 in path_separators)`.

Reply:
The mixed cases don't work in Explorer, and they're edge case enough that I'm comfortable saying they're undefined behaviours now.

Though I do now see that the cheap join function here won't work properly with a single leading separator, so that needs fixing.